server/server.go

```diff
@@ -0,0 +1,46 @@
+package server
+
+import (
+	"fmt"
+	mw "github.com/kirikami/go_exercise_api/middleware"
+	"github.com/kirikami/go_exercise_api/routes"
+	"github.com/labstack/echo"
+	"github.com/labstack/echo/engine/standard"
+	"github.com/labstack/echo/middleware"
+	"net/http"
+)
+
+func StartServer(app routes.ApiV1Handler) {
+	server := echo.New()
+
```

Comment:
server.Use(middleware.CORSWithConfig(middleware.CORSConfig{
 +		AllowHeaders: []string{echo.HeaderAccessControlAllowOrigin},
 +	}))
making it according to assignment